qiskit/visualization/matplotlib.py

```diff
@@ -757,9 +751,11 @@ def _draw_ops(self, verbose=False):
                 # add conditional
                 if op.condition:
                     cond_xy = [
-                        self._c_anchors[ii].plot_coord(this_anc, layer_width, self._x_offset)
+                        self._c_anchors[ii].plot_coord(anc_x_index, layer_width, self._x_offset)
                         for ii in self._clbits_dict
                     ]
+                    if self._clbits_dict:
+                        anc_x_index = self._c_anchors[0].get_x_index()
```

Comment:
This seems slightly worrying that `anc_x_index` is updated with a loop, and not used until the loop exits - it could in theory get overwritten by a subsequent loop iteration.  I guess that with the current form, we can only ever have one conditional gate in any given layer and those are the only things that can update `anc_x_index`, so this will be safe, but logically I think the operation should be
```python
if self._clbits_dict:
    anc_x_index = max(anc_x_index, self._c_anchors[0].get_x_index())
```
if `anc_x_index` is meant to represent the right-most point?  Or is that incorrect if there's a line break?

Reply:
You're right that in current code only 1 condition can be shown per layer. I suppose that could change in the future, so probably doing `max` is prudent. `anc_x_index` continues to increment through a line break. It's the `xpos` that comes back from `plot_coords` that will go back to the start after a line break. Done in https://github.com/Qiskit/qiskit-terra/pull/7545/commits/b5ad6460d395f3178f4fd0b79059c4815dbcc457.